API Blueprint Specification.md

```diff
@@ -478,6 +490,232 @@ Name and description of the API
+- **Markdown entity:** header
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authentication` keyword in Markdown header entity.
+
+    # Authentication
+
+#### Description
+Authentication settings for the whole API.
+
+This section **should** include one [Authentication schemes section](#def-authenticationschemes-section) and one [Response section](#def-response-section).
+
+The [Authentication schemes section](#def-authenticationschemes-section) defines all the types of authentication schemes supported by the API. The [Response section](#def-response-section) describes the error message returned in the case of failed authentication.
+
+##### Example
+
+    ## Authentication
+    This API supports different kinds of the authentication.
+
+    + Authentication Schemes
+        + ...
+
+    + Response 403 (application/json)
+
+            {
+              "message": "Authentication required",
+              "type": "error"
+            }
+
+---
+
+<a name="def-authenticationschemes-section"></a>
+## Authentication schemes section
+- **Parent sections:** [Authentication section](#def-authentication-section)
+- **Nested sections:** none
+- **Markdown entity:** list
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authentication Schemes` keyword.
+
+    + Authentication Schemes
+
+#### Description
+List of authentication schemes supported by the API.
+
+API Blueprint supports three **Base Authentication Schemes** (described below) which can be customized to derive authentication schemes supported by the API. Each of the base authentication schemes have a list of customizable options.
+
+Each authentication scheme is defined by an authentication scheme [name (identifier)](#def-identifier), base authentication scheme and the respective customized options.
+
+This section **must** be composed of nested list items only. This section **must not** contain any other elements. Each list item describes a single Authentication scheme.
+
+    + <scheme name> (<base scheme type>) - <description>
+
+        + <customizable option 1>
+        + <customizable option 2>
+        ...
+        + <customizable option N>
+
+Where:
+
++ `<scheme name>` is the authentication scheme name by which this scheme is represented in [Authenticated section](#def-authenticated-section).
++ `<description>` is any **optional** Markdown-formatted description of the authentication scheme.
++ `<base scheme type>` is the base authentication scheme type (e.g. "Basic Authentication Scheme"). See below for all the available base authentication schemes.
++ `<customizable option n>` represents a customizable option in the base authentication scheme. It is a [MSON Property Member Declaration][] without the [MSON Type Definition][] where the [MSON Property Name][] represents the name of the option and [MSON Value][] represents the value of the option.
```

Comment:
If you look at the description of parameters, `N` is an example while `n` is a generic.